Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.cpp

```diff
@@ -150,6 +151,17 @@ namespace AzToolsFramework
             return QDialogButtonBox::DestructiveRole;
         }
 
+        void PrefabSaveHandler::ExecuteSavePrefabDialog(AZ::EntityId entityId)
+        {
+            auto instanceEntityMapper = AZ::Interface<InstanceEntityMapperInterface>::Get();
+            AZ_Assert(instanceEntityMapper, "Prefab - PrefabSaveHandler - could not get InstanceEntityMapperInterface when saving prefab.");
```

Comment:
Could this be retrieved on initialization and be reused?

Also, please add "Interface" to the variable name for consistency
```
auto instanceEntityMapperInterface = AZ::Interface<InstanceEntityMapperInterface>::Get();
```

Reply:
I tried set this up in the constructor but it looks like the value changes to invalid value over time (after initialization). I'll do the renaming.

Reply:
hum that's weird, does the mapper get re-instantiated? We should investigate that.

Reply:
Interestingly, I made the changes and the issue was not showing up anymore. I manually tested it in editor and no issues showed up :)